### ISSUE TITLE
Weird 'routes' output 

### ISSUE THREAD

**Issue by adamlwatson** [cldwalker/tux#7]
On a Mac OS X machine running ruby 1.9.3p448, I'm seeing strange output when I issue the "routes" command. An example:

GET     "/\A\/submitjob\/:sleeptime\z/"
GET     "/\A\/ip\z/"
GET     "/\A\/endpoint\/?:day?\z/"

Any idea what could be going on, here?

**Comment by cldwalker** [comment]
Thanks for reporting your issue! This is [one of my 18 active issues](https://gh-active-issues.herokuapp.com/#cldwalker/tux#7). Use that link to check how soon your issue will be answered. Don't forget to check your issue against this project's CONTRIBUTING.md. Cheers.

**Comment by cldwalker** [comment]
Yeah, those are regular expression characters. The routes command works by [converting the regular expressions back to readable form](https://github.com/cldwalker/tux/blob/master/lib/tux/commands.rb#L5). I'm ok with the way it looks now but I'm open to a pull request that strips away these special characters from `regex.inspect`.
